polaris-react/src/components/Button/Button.scss

```diff
@@ -158,6 +159,11 @@
   + *:not(.Icon) {
     margin-left: var(--p-space-05);
   }
+
+  svg {
+    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+    fill: var(--pc-button-icon-color);
+  }
```

Comment:
Instead of this being declared over an over again, it's now declared once and the custom property is set in the class and interaction state declarations instead.

Reply:
(ignore)